sources/osg/PrimitiveFunctor.js

```diff
@@ -341,7 +341,7 @@ PrimitiveFunctor.prototype = {
         var primitives = geom.getPrimitiveSetList();
         if (!primitives) return;
 
-        var cb = this._cb();
+        var cb = this._cb;
```

Comment:
As far as the API changes the comments in the class should be adapted. Also, is this API better than before? why? 
Note that I'm not agains it, I only want to start to stabilize the API.

Reply:
yep, I'll update the comment.

I didn't see the point in using a function, the indirection wasn't necessary (main reason).
The other reason is that it simplifies the code elsewhere when we need to call this function, in that case we can simply provide an instance that have the 3 required fonction instead of creating a function + an object

Reply:
Ok my point is more that it would be nice to start to respect the API. So if we merge this it would be nice to be sure to stick with ithrough the time. Take into account that this change will affect users.

Reply:
Agreed.
I believe we can stick with just an object, if at some point we really need a function, we can just do `var cb = this._cb === 'function' ? this._cb() : this._cb`